services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -4538,7 +4571,7 @@ func TestTransformer(t *testing.T) {
 						t.Errorf("unexpected version: expected 3, actual %d", *prodVersion)
 					}
 					checkReleaseExists := func(v uint64) {
-						_, err := s.GetApplicationRelease("test", v)
+						_, err := s.GetApplicationReleaseFromManifest("test", v)
```

Comment:
Should we add tests that test for reading from the database? I would also suggest they don't be separate tests, but we use both in the same test and compare them.

Reply:
I wouldn't want to do this here. The tests here are not uniform at all (they're not really table driven tests).
I will add more tests to the transformer_db_test.go though.

Reply:
done, see transfomer_db_test.go, I added "expectedDbReleases"